mne/io/edf/edf.py

```diff
@@ -51,6 +51,8 @@ class RawEDF(BaseRaw):
     annotmap : str | None
         Path to annotation map file containing mapping from label to trigger.
         Must be specified if annot is not None.
+    exclude : array of str
+        Channel names to exclude.
```

Comment:
Add a note that this can help loading channels with higher sampling rates to avoid unnecessary resampling.